GoogleForCreators/web-stories-wp
# Design Panel: Improve accessibility for panel headers

## Summary

For sections 'Size & position', 'Layer', 'Style', 'Link'
1.disable navigation to each header's A11Y subtree
2.Add aria-expanded to each section header

## Relevant Technical Choices

Move the aria-* labels from the nested button to the header

## To-do

N/A

## User-facing changes

N/A

## Testing Instructions

Testing Instructions
Enable VoiceOver
Open Web Stories and choose a Shape Element
Use the VO key to navigate the header
Make sure the voice will announce the status of the section header and skip over the nested button.
Sample announcement text could be Size & position panel, expanded, button

---

Fixes #4603
